.circleci/config.yml

```diff
@@ -257,7 +257,52 @@ jobs:
           root: .
           paths:
             - ui
-
+  
+  puppeteer-e2e-test:
+    docker:
+      - image: circleci/node:12.16.1
+    working_directory: ~/workbench
+    steps:
+      - checkout
+      - run:
+          name: Fetch Submodules
+          command: git submodule update --init --recursive
+      - restore_cache:
+          keys:
+          - ui-cache-{{ checksum "~/workbench/e2e/yarn.lock" }}
```

Comment:
Please name this key differently here and below:

e.g.

```
- e2e-cache-{{ checksum ... }}
- e2e-cache-
```

This is not caching the same thing as the above UI tests